.ddev/config.local.yaml.example

```diff
@@ -7,6 +7,14 @@ hooks:
         > 0 AND mail LIKE '%~%'"
     # This will be applied to production with a future release.
     - exec: drush sqlq "DELETE FROM system WHERE name='hedley_pusher'"
+    - exec: drush updb -y
+    - exec: drush updb -y
+    # Modules can come and go, production might be behind.
+    - exec: wget https://ftp.drupal.org/files/projects/registry_rebuild-7.x-2.5.tar.gz
+    - exec: tar -xvf registry_rebuild-7.x-2.5.tar.gz --directory ~/.drush
+    - exec: rm registry_rebuild-7.x-2.5.tar.gz
+    - exec: drush rr
```

Comment:
for me this line failed when it is run before the cache clear.

Reply:
@AronNovak ^^

Reply:
right, adjusted